Guessing numbers.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+This is a search task in which participants guess numbers between 0-100 and the computer tells them how many point were obtained from the guess. The participants were told their guess and scores as well as the guesses and scores of their neighbors so that the participants could choose to imitate high-scoring guesses from their peers.
+
+# References
+Main paper: https://www.researchgate.net/publication/23196347_Propagation_of_Innovations_in_Networked_Groups
+
```

Comment:
Small world network structure:, Ahmed E. & Abdusalam, H. A. (2000). On social percolation and small world network. European Physical Journal B, 15, 569-571.